modules/op-resolver-stepactions-ref.adoc

```diff
@@ -0,0 +1,79 @@
+// This module is included in the following assemblies:
+// * create/remote-pipelines-tasks-resolvers.adoc
+
+// // *openshift_pipelines/remote-pipelines-tasks-resolvers.adoc
+:_mod-docs-content-type: REFERENCE
+[id="resolver-stepactions-ref_{context}"]
+= Step action provided with {pipelines-shortname}
+
+{pipelines-shortname} provides a standard `StepAction` definition that you can use in your tasks. Use the HTTP resolver to reference this definition.
+
+[discrete]
+[id="op-stepaction-git-clone_{context}"]
+== git-clone
+
+The `git-clone` step action uses Git to initialize and clone a remote repository on a workspace. You can use this step action to define a task that clones a repository at the start of a pipeline that builds or otherwise processes this source code.
+
+To reference this step action in a task, use the HTTP resolver with the following URL: `\https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml`.
```

Comment:
Long URLs might work better as their own code block (plus you get a copy/paste button):
```suggestion
To reference this step action in a task, use the HTTP resolver with the following URL: 

[source,text]
----
https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml
----
```